modelexpress_common/proto/refit.proto

```
@@ -82,40 +82,37 @@ message WeightVersion {
   // Reserved placeholder for a future server-assigned layout fingerprint.
   // Content identity currently travels in the payload.
-  string layout_signature = 8;
-  WeightVersionState state = 9;
-  uint64 created_at_unix_ms = 10;
+  string layout_signature = 7;
+  WeightVersionState state = 8;
+  uint64 created_at_unix_ms = 9;
   // Present only when the version payload is published through object storage.
-  ObjectStorageSource object_storage = 11;
+  ObjectStorageSource object_storage = 10;
 }
```

Buf (1.72.0)

[error] Previously present field "11" with name "object_storage" on message "WeightVersion" was deleted without reserving the name "object_storage".

(FIELD_NO_DELETE_UNLESS_NAME_RESERVED)